### PR TITLE
Feat/utilities

### PR DESCRIPTION
**Pull request by Aditi0505**
# Change Logs:

## Description:

- Added current time in sync
- Added Quotes. Quotes change on refresh
- Added current location and current weather

## Screenshot 
image

### CODE REVIEW

File: src/context/quote-context.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get(
+          "https://api.quotable.io/random?maxLength=100"
+        );
+        setQuote(response.data.content);
+      } catch (e) {
+        console.log(e);
```
Comment (prakashsakari) [inline src/context/quote-context.js:16]: you can remove this log before merge

File: src/context/weather-context.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        const response = await axios.get(
+          `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=${key}`
+        );
+        weatherDetailsDispatch({
+          type: "ADD_WEATHER_DETAILS",
+          payload: response,
+        });
+      })();
+    } catch (e) {
+      console.log(e);
```
Comment (prakashsakari) [inline src/context/weather-context.js:44]: remove the log from here also

**Review by prakashsakari — APPROVED** [review]
UI looks clean
Code is readable
Good use of reducer
Good to merge

**Review by hiteshverma27 — APPROVED** [review]
code looks good
good folder structure
I like how minimal the code is

**Pull request merged**